core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -127,6 +128,9 @@ CFG_WDT ?= $(CFG_STM32_IWDG)
 # Platform specific configuration
 CFG_STM32MP_PANIC_ON_TZC_PERM_VIOLATION ?= y
 
+# Default disable shared resources
+CFG_STM32MP1_SHARED_RESOURCES ?= n
```

Comment:
as CFG_STM32MP13=y does not expect to enable `CFG_STM32MP1_SHARED_RESOURCES`, I would suggest to force it disabled in bertween lines 69 to 75:
```patch
 ifeq ($(CFG_STM32MP13),y)
 $(call force,CFG_STM32MP15,n)
 $(call force,CFG_STM32MP_CLK_CORE,y)
+$(call force,CFG_STM32MP1_SHARED_RESOURCES,n)
 $(call force,CFG_STM32MP13_CLK,y)
 $(call force,CFG_STM32MP15_CLK,n)
 CFG_STM32MP_OPP_COUNT ?= 2
 else
```

Reply:
> as CFG_STM32MP13=y does not expect to enable `CFG_STM32MP1_SHARED_RESOURCES`, I would suggest to force it disabled in bertween lines 69 to 75:
> 
> ```diff
>  ifeq ($(CFG_STM32MP13),y)
>  $(call force,CFG_STM32MP15,n)
>  $(call force,CFG_STM32MP_CLK_CORE,y)
> +$(call force,CFG_STM32MP1_SHARED_RESOURCES,n)
>  $(call force,CFG_STM32MP13_CLK,y)
>  $(call force,CFG_STM32MP15_CLK,n)
>  CFG_STM32MP_OPP_COUNT ?= 2
>  else
> ```


It is disabled by default but I guess we can force it to prevent further issues

Reply:
remove these lines since config switch is forced above.